### PR TITLE
Stoneintg 477

### PR DESCRIPTION
**Pull request by jencull**
amended to one commit with extended commit message

### CODE REVIEW

File: 95a50c7 [hunk 1]
```diff
@@ -0,0 +1,45 @@
+[33mfbdb95c[m[33m ([m[1;36mHEAD -> [m[1;32mmain[m[33m, [m[1;31morigin/main[m[33m, [m[1;31morigin/HEAD[m[33m)[m HEAD@{0}: reset: moving to HEAD
```
Comment (MartinBasti) [inline 95a50c7:1]: What is this?

Comment (MartinBasti) [reply 95a50c7:1]: I mean why git reflog is committed?

Comment (jencull) [reply 95a50c7:1]: I need to add something to my .gitignore to prevent this?

Comment (MartinBasti) [reply 95a50c7:1]: I don't know how you get that into commit, it looks like regular file. Have you accidentaly saved git reflog into file?

Comment (MartinBasti) [reply 95a50c7:1]: `.git/` folders are ignored by default

File: .gitignore [hunk 2]
```diff
@@ -22,4 +22,4 @@ vendor
 .idea
 *.swp
 *.swo
-*~
+*~
```
Comment (MartinBasti) [inline .gitignore:25]: missing newline at the end

Comment (jencull) [reply .gitignore:25]: ok

**Comment by MartinBasti** [comment]
A unittest is failing

**Comment by jencull** [comment]
> A unittest is failing

that was a timing issue and an eventually block has been added to work this out.

File: loader/loader_test.go [hunk 3]
```diff
@@ -363,6 +371,21 @@ var _ = Describe("Loader", Ordered, func() {
 			},
 		}
 		Expect(k8sClient.Create(ctx, hasBinding)).Should(Succeed())
+		//Jen
```
Comment (MartinBasti) [inline loader/loader_test.go:374]: forgotten comment?

Comment (jencull) [reply loader/loader_test.go:374]: bookmark! removed now.

**Review by MartinBasti — APPROVED** [review]

**Review by MartinBasti — CHANGES_REQUESTED** [review]
There are 2 commits, please squash it into one

**Review by MartinBasti — APPROVED** [review]

**Review by dirgim — APPROVED** [review]
LGTM

**Pull request merged**